src/server/ua_subscription.h

```diff
@@ -215,6 +219,7 @@ UA_StatusCode Subscription_registerPublishCallback(UA_Server *server, UA_Subscri
 UA_StatusCode Subscription_unregisterPublishCallback(UA_Server *server, UA_Subscription *sub);
 void UA_Subscription_addMonitoredItem(UA_Subscription *sub, UA_MonitoredItem *newMon);
 UA_MonitoredItem * UA_Subscription_getMonitoredItem(UA_Subscription *sub, UA_UInt32 monitoredItemId);
+UA_MonitoredItem * UA_Subscription_checkMonitoredItem(UA_Subscription *sub, UA_NodeId node);
```

Comment:
This method is not implemented and not used in the PR.

Reply:
The function is deprecated for this PR and will be removed.